message_ix/model/MESSAGE/sets_maps_def.gms

```diff
@@ -202,11 +202,9 @@ Alias(mode, mode2);
 *    * - type_tec_land (type_tec) [#type_tec_land]_
 *      - :math:`\widehat{t} \in \widehat{T}^{LAND} \subseteq \widehat{T}`
 *      - Mapping set of technology types and land use
-*    * - commodity_surplus (commodity)
-*      - :math:`c \in C^{FULL} \subseteq C`
-*      - Commodities related to `Equation COMMODITY_BALANCE_GT`_
-*    * - commodity_shortage (commodity)
+*    * - balance_equality (commodity,level)
 *      - :math:`c \in C^{FULL} \subseteq C`
+*      - :math:`l \in L^{FULL} \subseteq L`
 *      - Commodities related to `Equation COMMODITY_BALANCE_LT`_
```

Comment:
if this is related to the mapping set `balance_equality`, please write accordingly

Reply:
of course it's related to the set `balance_equality`, it is its description. I also don't understand this comment, I will just add `level` to the description, if it's ok